### PR TITLE
update(KeyBinds): Create shortcut to open uplink when it is minimized

### PR DESCRIPTION
**Pull request by lgmarchi**
### 1. Create shortcut to open uplink when it is minimized

https://github.com/Satellite-im/Uplink/assets/63157656/605f3d42-f5f2-4813-8a14-1cf99d446205

### Which issue(s) this PR fixes 🔨

- Resolve #

### CODE REVIEW

**Comment by phillsatellite** [comment]
@lgmarchi Hey bro, so on Mac the maximize is working great but on windows I wasnt able to get it to work 

https://github.com/Satellite-im/Uplink/assets/93608357/7321e1e7-33cb-4335-8bc6-a05dc4452fb3

**Comment by phillsatellite** [comment]
Lucas is taking a look into an issue I DM'd him about, if you are clicked into another app like Chrome and then you use the shortcut to maximize Uplink you will see Uplink flash open for a minute but it will get hidden behind Chrome 

https://github.com/Satellite-im/Uplink/assets/93608357/7c6e02b5-16a7-445e-88b6-a2f2edee9eb2

**Review by phillsatellite — APPROVED** [review]

File: common/locales/en-US/main.ftl [hunk 1]
```diff
@@ -299,6 +299,7 @@ settings-keybinds = Keybind Settings
     .open-dev-tools = Open Web Inspector
     .toggle-devmode = Toggle Developer Mode
     .cancel-change-keybind = Cancel Recording
+    .set-app-visible = Set Uplink Visible
```
Comment (InfamousVague) [inline common/locales/en-US/main.ftl]: Small change before we merge, can we change this to "Open/Focus Uplink"

**Review by InfamousVague — APPROVED** [review]
Small change requested then it should be good to go!

**Review by InfamousVague — APPROVED** [review]

**Pull request merged**